File: Program39..java
import java.lang.*;
import java.util.*;
class Prime
{
	public int iNo;
	public Prime(int iValue)
	{
		iNo=iValue;
	} 
	public boolean ChckPrime()
	{
		int iCnt=0;
		boolean bFlag=true;
		if(iNo<0)
		{
			iNo=(-iNo);
		}
		for (iCnt=2; iCnt<=(iNo/2);iCnt++)
		{
			if((iNo%iCnt)==0)
			{
				bFlag=false;
				break;
			}
		}
		return bFlag;
	}
}
class Program39
{
	public static void main(String a[])
	{
		int iValue=0;
		boolean bRet=false;
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter Number");

		iValue=sobj.nextInt();

		Prime pobj=new Prime(iValue);

		bRet=pobj.ChckPrime();

		if(bRet==true)
		{
			System.out.println(iValue+" is a prime number");
		}
		else 
		{
			System.out.println(iValue+" is not a prime number");
		}
	}
}

File: program39.c
#include <stdio.h>
#include<stdbool.h>

bool CheckPrime(int iNo)
{
	int iCnt=0;
	bool bFlag=true;

	if(iNo<0)
	{
		iNo=(-iNo);
	}

	for(iCnt=2; iCnt<=(iNo/2); iCnt++)
	{

		if(iNo%iCnt==0)
		{
			bFlag=false;
			break;
		}
		
	}
	return bFlag;
}

int main()
{
	int iValue=0;
	bool bRet=false;
	printf("Enter number: ");
	scanf("%d",&iValue);

	bRet=CheckPrime(iValue);
	if(bRet==true)
	{
	printf("%d is a prime number\n",iValue);
	}
	else
	{
		printf("%d is not a prime number\n",iValue);
	}
	return 0;
}